projects/1.21-fabric/assets/newworld/newworld/lang/zh_cn.json

```
@@ -0,0 +1,86 @@
+{
+  "advancements.adventure.collect_ancient_mattock.description": "使用模板碎片合成一个远古鹤嘴锄",
+  "advancements.adventure.collect_ancient_mattock.title": "复兴的工具",
+  "advancements.adventure.collect_template_fragment.description": "发现一个模板碎片",
+  "advancements.adventure.collect_template_fragment.title": "一段历史",
+  "biome.newworld.wooded_meadow": "繁茂的草甸",
+  "block.newworld.fir_button": "冷杉按钮",
+  "block.newworld.fir_door": "冷杉门",
+  "block.newworld.fir_fence": "冷杉栅栏",
+  "block.newworld.fir_fence_gate": "冷杉栅栏门",
+  "block.newworld.fir_leaves": "冷杉树叶",
+  "block.newworld.fir_log": "冷杉原木",
+  "block.newworld.fir_planks": "冷杉木板",
+  "block.newworld.fir_pressure_plate": "冷杉压力板",
+  "block.newworld.fir_sapling": "冷杉树苗",
+  "block.newworld.fir_slab": "冷杉台阶",
+  "block.newworld.fir_stairs": "冷杉楼梯",
+  "block.newworld.fir_trapdoor": "冷杉活板门",
+  "block.newworld.fir_wood": "冷杉木头",
+  "block.newworld.loam": "壤土",
+  "block.newworld.loam_brick_slab": "壤土砖台阶",
+  "block.newworld.loam_brick_stairs": "壤土砖楼梯",
+  "block.newworld.loam_brick_wall": "壤土砖墙",
+  "block.newworld.loam_bricks": "壤土砖块",
```

这个方块有无合成所需的砖物品，比如红砖之类的
没有请改为
```suggestion
  "block.newworld.loam_bricks": "壤土砖",
```
与原版石砖相同